content/departments/product-engineering/engineering/cloud/devops/managed/creation_process.md

```diff
@@ -43,23 +47,24 @@ For basic operations like accessing an instance for these steps, see [managed in
    - Email: `managed+$COMPANY@sourcegraph.com` (note `+` sign not `-`)
    - Username: `sourcegraph-admin`
    - Password: Use the password previously created and stored in 1password.
-1. Create a token for the account under `/users/sourcegraph-admin/settings/tokens` called `managed-instances` and add it as "token" under the 1password entry for the admin account.
-1. Navigate to Grafana and confirm the instance looks healthy.
+1. Create a token for the account under `/users/sourcegraph-admin/settings/tokens` called `managed-instances` with `site-admin:sudo` checked and add it as "token" under the 1password entry for the admin account.
+1. Navigate to Grafana under `/-/debug/grafana` and confirm the instance looks healthy.
 1. Configure `externalURL` in the site configuration, and use SSH to restart the server (`sudo su`, `shutdown -r`) wait for it to come back up and access it again.
```

Comment:
@michaellzc @mucles is there any reason to reboot the entire machine, or could this be simplified to just a restart of the `frontend` container?

Reply:
great question. I also wonder why do we need to restart the entire VM...

I have never done any MI creation yet, @mucles what do you usually do?

Reply:
What is understand this is same as [restart for configuration change](https://handbook.sourcegraph.com/departments/product-engineering/engineering/cloud/devops/managed/operations/#restarting-for-configuration-updates) in handbook?

Reply:
@sanderginn @michaellzc - Since it is just a site config changes the same restart process in the handbook that @filiphaftek  mentioned, should do the same thing I don't think there is a need to restart the entire VM. Mind you though I haven't tried it in practice when I have done the creations I have always used the `shutdown -r `command